mailqueue/tasks.py

```diff
@@ -8,7 +8,18 @@
 @shared_task(name="tasks.send_mail", default_retry_delay=5, max_retries=5)
 def send_mail(pk):
     message = MailerMessage.objects.get(pk=pk)
-    message._send()
+    try:
+        message._send()
+    except Exception as e:
+        # Assuming that Django project uses anymail.backends.postmark.EmailBackend
+        if e.__class__.__name__ == "AnymailRecipientsRefused":
```

Comment:
Logic here is relying on the fact that at ZPD, at the moment we're using combination of
`django-mail-queue` and `django-anymail`. Still, I decided to use string class name of `AnymailRecipientsRefused ` exception here instead of importing it. So that we'll not have tight coupling between these 3rd party packages.  
The code here does not look perfect for long term. But we don't know if we'll use `django-mail-queue` forever or not. So, I think for now this is the most practical solution.